tests/unit/DebugUtilsTest.ts

```diff
@@ -1562,17 +1507,23 @@ describe('DebugUtils', () => {
                         reportID: '1',
                         parentReportActionID: '0',
                         stateNum: CONST.REPORT.STATE_NUM.OPEN,
-                        statusNum: CONST.REPORT.STATE_NUM.SUBMITTED,
+                        ownerAccountID: 1234,
+                        policyID: '1',
+                    },
+                    [`${ONYXKEYS.COLLECTION.TRANSACTION}0` as const]: {
```

Comment:
```suggestion
                    [`${ONYXKEYS.COLLECTION.TRANSACTION}1` as const]: {
```

Shouldn't the transactionID in the onyx key match the one in the value?

Reply:
Ah it does not matter for the test, but yes, that would be more correct